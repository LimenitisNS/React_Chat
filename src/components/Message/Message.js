import React from "react";
import styles from "./styles.module.css";

export default function Message({ content, nickname }) {
  return (
    <div className={styles.message}>
      <li>
        <b>{nickname}:</b>
        {content}
      </li>
    </div>
  );
}
