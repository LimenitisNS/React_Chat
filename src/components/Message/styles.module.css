.message{
    font-size: 20px;
    list-style-type: none;
    word-break:break-all;
}