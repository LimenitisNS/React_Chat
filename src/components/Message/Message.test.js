import React from "react";
import { shallow } from "enzyme";
import Message from "./Message";
import renderer from "react-test-renderer";

describe("Message", () => {
  test("Message shows nick and message", () => {
    const nick = "test";
    const message = "test";
    const component = shallow(<Message nick={nick} message={message} />);
    expect(component.text()).toContain(nick);
    expect(component.text()).toContain(message);
  });

  test("Snaphot", () => {
    const message = "test";
    const component = renderer.create(<Message message={message} />);
    let componentSnapshot = component.toJSON();
    expect(componentSnapshot).toMatchSnapshot();
  });
});
