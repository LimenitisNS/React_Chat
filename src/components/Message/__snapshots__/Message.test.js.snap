// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Snaphot 1`] = `
<div>
  <li>
    <b>
      :
    </b>
  </li>
</div>
`;
